scripts/cat/cats.py

```diff
@@ -2927,133 +2890,21 @@ def mediate_relationship(mediator, cat1, cat2, allow_romantic, sabotage=False):
-                        rel1.respect,
-                        -(randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                    rel2.respect = Cat.effect_relation(
-                        rel2.respect,
-                        -(randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                else:
-                    rel1.respect = Cat.effect_relation(
-                        rel1.respect,
-                        (randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                    rel2.respect = Cat.effect_relation(
-                        rel2.respect,
-                        (randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
+            ran = (5, 10) if value == RelType.ROMANCE and mates else (4, 6)
 
-            elif value == RelValue.COMFORT:
-                ran = (4, 6)
-
-                if sabotage or value in chosen_neg:
-                    rel1.comfort = Cat.effect_relation(
-                        rel1.comfort,
-                        -(randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                    rel2.comfort = Cat.effect_relation(
-                        rel2.comfort,
-                        -(randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                else:
-                    rel1.comfort = Cat.effect_relation(
-                        rel1.comfort,
-                        (randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                    rel2.comfort = Cat.effect_relation(
-                        rel2.comfort,
-                        (randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-
-            elif value == RelValue.TRUST:
-                ran = (4, 6)
-
-                if sabotage or value in chosen_neg:
-                    rel1.trust = Cat.effect_relation(
-                        rel1.trust,
-                        -(randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                    rel2.trust = Cat.effect_relation(
-                        rel2.trust,
-                        -(randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                else:
-                    rel1.trust = Cat.effect_relation(
-                        rel1.trust,
-                        (randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
-                    rel2.trust = Cat.effect_relation(
-                        rel2.trust,
-                        (randint(ran[0], ran[1]) + bonus) + personality_bonus,
-                    )
+            change = ((randint(ran[0], ran[1]) + bonus) + personality_bonus) * (
+                -1 if sabotage else 1
+            )
 
-                decrease = not decrease
+            setattr(rel1, value, Cat.effect_relation(getattr(rel1, value), change))
```

Comment:
`effect_relation` should be `affect_relation`. 

and it's super weird that that function doesn't handle the affecting of the actual relationship, but i digress. Could consider renaming?